toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -36,7 +40,18 @@ func convertToCosi(ic *ImageCustomizerParameters) error {
 	}
 	defer imageLoopback.Close()
 
-	partitionMetadataOutput, err := extractPartitions(imageLoopback.DevicePath(), outputDir, ic.outputImageBase, "raw-zst", ic.imageUuid)
+	// Get partition info
+	diskPartitions, err := diskutils.GetDiskPartitions(imageLoopback.DevicePath())
+	if err != nil {
+		return err
+	}
+
+	roothash, rootPartUUID, roothashFilepath, err := locateVerityRootInfo(diskPartitions, outputDir, ic.outputImageBase)
```

Comment:
Instead of trying to extract the verity info from the final image, please plumb the verity info from the `customizeVerityImageHelper` function (which is where the verity info is generated) to the `convertToCosi` function.

Extracting the verity info from the image correctly is a bit of a pain. This is also needed by PR #83. However, PR #83 currently handles extracting the info from UKIs, which this change is missing. Therefore, it is probably easier to just take the verity info from the place that it is being generated.